deploy/buildtools/buildconfig.py

```diff
@@ -108,6 +108,12 @@ def __init__(self, name: str, recipe_config_dict: Dict[str, Any], build_farm_hos
 
         self.build_farm_host_dispatcher.parse_args()
 
+        self.fpga_bit_builder_dispatcher_class_name = recipe_config_dict.get('fpga-platform', "F1BitBuilder")
+        # create run platform dispatcher object using class given and pass args to it
+        self.fpga_bit_builder_dispatcher = getattr(
+            import_module("buildtools.bitbuilder"),
+            self.fpga_bit_builder_dispatcher_class_name)(self)
```

Comment:
Why do you use the raw python class names for BitBuilder classes but for BuildHostDispatcher, you use a different string name and build up a mapping?  I would expect that you would want to use the same pattern in both places.

Reply:
Also, I think that we want to be able to plugin other bitbuilders without having to pollute the firesim codebase.  We should move away from only being able to lookup BitBuilderDispatchers in the buildtools.bitbuilder module.

Reply:
I didn't come across this while merging and getting things to work again.  We probably still want to make the construction of the `fpga_bit_builder_dispatcher_class_name` work the same way as the other places where we're using the inheritors map.  Since that code is moving to `deploy/utils/inheritors.py` in #1028, perhaps we should merge this as is and clean it up to be consistent with the new code location in #1028?

Reply:
That PR will take longer to merge in so I think this plan is good (merge this 1st - cleanup in the other PR)